stdlib/universe/map.go

```diff
@@ -114,7 +115,8 @@ func createMapTransformation(id execute.DatasetID, mode execute.AccumulationMode
 	}
 	cache := execute.NewTableBuilderCache(a.Allocator())
 	d := execute.NewDataset(id, mode, cache)
-	t, err := NewMapTransformation(d, cache, s)
+	t, err := NewMapTransformation(d, cache, s, a.Context(), a.Dependencies()["interpreter"].(dependencies.Interface))
```

Comment:
we should be able to change this to executetest.NewTestExecuteDependencies()

Reply:
no, this isn't test code.